epsagon/events/dbapi.py

```diff
@@ -108,7 +108,12 @@ def __init__(
                 (operation == 'select') or
                 (not trace_factory.metadata_only)
         ):
-            self.resource['metadata']['Query'] = query[:MAX_QUERY_SIZE]
+
+            query_max = query[:MAX_QUERY_SIZE]
+            if trace_factory.obfuscate_sql:
+                query_max = obfuscate_sql_query(query_max, operation)
```

Comment:
lets wrap this call with try except and in case of failures use print_debug with a relevant message